### ISSUE TITLE
Rule Proposal: Recommend using count() instead of length

### ISSUE THREAD

**Issue by alecxe** [alecxe/eslint-plugin-protractor#53]
Invalid usage:

``` js
element.all(by.repeater('product in products')).then(function(products){
    expect( products.length >= 1 ).toBeTruthy();
});
```

Valid usage:

``` js
expect(element.all(by.repeater('product in products')).count()).toBeGreaterThan(1);
```

Related: http://stackoverflow.com/questions/39043554/pulling-ng-repeat-array-in-protractor

**Comment by alecxe** [comment]
Closed by 392731372877221a3323917bc50d394094a041ad.
